src/cpu.c

```diff
@@ -70,8 +70,9 @@ unsigned int indirectYAddressMode(struct CPU *cpu, struct BUS *bus) {
 }
 
 unsigned int relativeAddressMode(struct CPU *cpu, struct BUS *bus) {
-  unsigned int address = cpu->pc + (signed char)readBus(cpu->pc, bus); 
+  signed char base = (signed char)readBus(cpu->pc, bus);
```

Comment:
Is this the fix that you mentioned yesterday, right?

Reply:
Exactly, you need to increment the PC before sum.